docs/first-steps/modeling/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Running your first data models"
+sidebar_position: 5
+sidebar_label: "Modeling data"
+description: "Using a data model to aggregate your data"
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+Querying the events table directly — as you would have done in the previous step — can be useful for exploring your events or building custom analytics. However, for many common use cases it’s much easier to use our [data models](/docs/modeling-your-data/modeling-your-data-with-dbt/index.md), which provide a pre-aggregated view of your data.
+
+We recommend [dbt](https://www.getdbt.com/) for data modeling. Here’s how to get started.
+
+<Tabs groupId="offering" queryString>
+  <TabItem value="enterprise" label="BDP Enterprise" default>
+
+Refer to the [setup instructions](/docs/modeling-your-data/running-data-models-via-snowplow-bdp/dbt/using-dbt/index.md) to add and configure your models in the Console, so that they can be run automatically by Snowplow BDP.
+
+  </TabItem>
+  <TabItem value="cloud" label="BDP Cloud">
+
+You will need to install dbt and run the models yourself — see the “quick start” links below.
+
+  </TabItem>
+  <TabItem value="try" label="Try Snowplow">
+
+You will need to install dbt and run the models yourself — see the “quick start” links below.
+
+  </TabItem>
+  <TabItem value="opensource" label="Open Source">
+
+You will need to install dbt and run the models yourself — see the “quick start” links below.
+
+  </TabItem>
+</Tabs>
+
+Next, add your first model:
```

Comment:
I think one thing that confuses people new to dbt is they should have their _own_ dbt project and add our packages to it. This project is what gets used by console etc. Might be worth trying to phrase that in here somewhere?